.github/dependabot.yml

```diff
@@ -25,21 +25,87 @@ updates:
+      - /packages/camera/camera_android/example/android/app
+      - /packages/camera/camera_android_camerax/example/android/app
+      - /packages/camera/camera/example/android/app
+      - /packages/espresso/example/android/app
+      - /packages/extension_google_sign_in_as_googleapis_auth/example/android/app
+      - /packages/file_selector/file_selector/example/android/app
+      - /packages/file_selector/file_selector_android/example/android/app
+      - /packages/flutter_adaptive_scaffold/example/android/app
+      - /packages/flutter_image/example/android/app
+      - /packages/flutter_markdown/example/android/app
+      - /packages/flutter_plugin_android_lifecycle/example/android/app
+      - /packages/go_router/example/android/app
+      - /packages/google_maps_flutter/google_maps_flutter/example/android/app
+      - /packages/google_maps_flutter/google_maps_flutter_android/example/android/app
+      - /packages/google_sign_in/google_sign_in/example/android/app
+      - /packages/google_sign_in/google_sign_in_android/example/android/app
+      - /packages/in_app_purchase/in_app_purchase_android/example/android/app
+      - /packages/in_app_purchase/in_app_purchase/example/android/app
+      - /packages/interactive_media_ads/example/android/app
+      - /packages/image_picker/image_picker/example/android/app
+      - /packages/image_picker/image_picker_android/example/android/app
+      - /packages/local_auth/local_auth_android/example/android/app
+      - /packages/local_auth/local_auth/example/android/app
+      - /packages/palette_generator/example/android/app
+      - /packages/path_provider/path_provider/example/android/app
+      - /packages/path_provider/path_provider_android/example/android/app
+      - /packages/platform/example/android/app
+      - /packages/pigeon/platform_tests/test_plugin/example/android/app
+      - /packages/pigeon/platform_tests/alternate_language_test_plugin/example/android/app
+      - /packages/quick_actions/quick_actions_android/example/android/app
+      - /packages/quick_actions/quick_actions/example/android/app
+      - /packages/rfw/example/hello/android/app
+      - /packages/rfw/example/local/android/app
+      - /packages/rfw/example/remote/android/app
+      - /packages/shared_preferences/shared_preferences/example/android/app
+      - /packages/shared_preferences/shared_preferences_android/example/android/app
+      - /packages/url_launcher/url_launcher_android/example/android/app
+      - /packages/url_launcher/url_launcher/example/android/app
+      - /packages/video_player/video_player/example/android/app
+      - /packages/video_player/video_player_android/example/android/app
+      - /packages/webview_flutter/webview_flutter/example/android/app
+      - /packages/webview_flutter/webview_flutter_android/example/android/app
+      - /third_party/packages/flutter_svg/example/android/app
     commit-message:
-      prefix: "[animations]"
+      prefix: "[gradle]"
```

Comment:
@stuartmorgan @reidbaker This change would mean we lose the PR title prefix, which is unfortunate. This seems worth it, but maybe some tooling/bot is relying on this prefix matching a package name?

Reply:
> maybe some tooling/bot is relying on this prefix matching a package name?

Shouldn't be, the prefixing convention in the repo is just for our convenience when wading through PRs and issues.